KenticoCloud.Delivery/KenticoCloud.Delivery.csproj

```diff
@@ -14,6 +13,7 @@
     <RepositoryType>git</RepositoryType>
     <PackageTags>kentico;mvc;aspnet;aspnetmvc</PackageTags>
     <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
```

Comment:
i'd also set [PublishRepositoryUrl ](https://github.com/dotnet/sourcelink#using-sourcelink) to true

Reply:
I have add PublishRepositoryUrl to the csprojs. But the nuspec already had the url un the repository tag.
* 6bdcc37

Reply:
Do we want to  leave it here?

Reply:
I'd leave it as it is now